### ISSUE TITLE
test against multiple versions of Python

### ISSUE THREAD

**Issue by jacquestardie** [recombinators/worker#27]
We know that this works well enough on Python 2.7, but what about other versions? Whether this is the only version that works or not, it would be useful to know definitively what does and doesn't work.

Looks like people are typically using [tox](https://tox.readthedocs.org/en/latest/) to sort that out.

![](http://media.giphy.com/media/yw6YcXlM895Nm/giphy.gif)

(I need to rebuild my gif collection)

**Comment by constanthatz** [comment]
Good idea. We haven't made a specific effort to make the whole thing Python 3.0 compatible, we should probably do that.
